hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1829,7 +1829,6 @@ private void updateActiveSnapshotMetrics()
         }
       }
     }
-
```

Comment:
nit: can you please revert this white line change?

Reply:
Fixed.